01-Getting-Started/main.tf

```diff
@@ -27,8 +27,13 @@
 provider "google" {
   project = var.project_id
   region  = var.region
-  version = "~> 3.53"
 }
+terraform {
+  required_providers {
+    google = {
+      version = "~> 3.53"
```

Comment:
Would be great if we can bump this to the latest 4.0 provider. This may also need some associated module version bumps.

Reply:
Yes, after a quick spot check it appears most module versions will need to be updated to support >3.53